vpr/src/draw/draw.cpp

```diff
@@ -1689,7 +1689,7 @@ static void draw_rr_edges(int inode, ezgl::renderer* g) {
     from_ptc_num = rr_graph.node_ptc_num(RRNodeId(inode));
 
     for (t_edge_size iedge = 0, l = device_ctx.rr_nodes[inode].num_edges(); iedge < l; iedge++) {
-        to_node = device_ctx.rr_nodes[inode].edge_sink_node(iedge);
+        to_node = size_t(rr_graph.edge_sink_node(RRNodeId(inode), iedge));
```

Comment:
I just read the codes of this function. It seems that we can eliminate the use of type casting ``RRNodeId(inode)`` and ``RRNodeId(to_node)``. 
In LINE 1675, you can see the ``rr_node`` variable is already defined.
In LINE 1678, you can change the data type of ``to_node`` from ``int`` to ``RRNodeId``

Then in the rest of this function, you just replace 
- ``RRNodeId(inode)`` -> ``rr_node``
- ``RRNodeId(to_node)`` -> ``to_node``.

We can do it in a follow-up PR.

Let me know what you think.

Reply:
But if we see at 
 https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/026c13c532200d7f498591ce5fd7e51db622e496/vpr/src/draw/draw.cpp#L1725
here in this function argument to_node  is as int type. Still `RRNodeId(to_node) -> to_node` Can be done ?

Reply:
@umariqbal-rs You can try ``draw_pin_to_pin(inode, size_t(to_node), g); ``

Reply:
@umariqbal-rs Can you let me know how many times we have to do the type casting? If it is too much, we can stay with what we have.

Reply:
@tangxifan 
Type casting with `size_t(to_node)` is more than the `RRNodeId(to_node)` so We should stay with the what we have.
Or We should do ?

Reply:
We can stay with what we have now.

Reply:
So should  I go on running QoR tests?